CONTRIBUTING.md

```diff
@@ -95,36 +98,33 @@ The HTMl report can be accessed at `htmlcov/index.html`.
 
 ## Documentation
 
-We write our documentation in [MyST Markdown][myst] for use in Sphinx. This is mainly stored in the `docs` folder,
-unless it's more appropriate to store it elsewhere, like this file.
-
-Further information on how to write Sphinx documentation, and how to build it into a searchable website can be found
-[here][docs-write-sphinx-documentation].
-
-## HM Government department frameworks
+We write our documentation in [MyST Markdown][myst] for use in Sphinx. This is mainly
+stored in the `docs` folder, unless it's more appropriate to store it elsewhere, like
+this file.
 
-HM Government frameworks are stored in the `.govcookiecutter/department_frameworks` folder. If you would like to add
-your own department's framework, follow the [instructions][docs-govcookiecutter-frameworks] in the `README.md` file in
-that folder.
+Further information on how to write Sphinx documentation, and how to build it into a
+searchable website can be found [here][docs-write-sphinx-documentation].
```

Comment:
Avoid 'here' as link text, for accessibility reasons. Some assistive tools put the links at the end of the document and "here" doesn't carry over the original context.

Reply:
Good call, made the changes (and others) in commit [`178de00`](https://github.com/ukgovdatascience/govcookiecutter/pull/31/commits/178de0063a87f15b35fed7fbb8052cb81d12dbf9). Will do an [accessibility test similar to GOV.UK Developer Docs](https://docs.publishing.service.gov.uk/accessibility.html#preparation-of-this-accessibility-statement) at a later date!